src/components/icons/SVGWrapper/SVGWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode, forwardRef } from "react";
+import { svgWrapper, SCVWrapperVaraints } from "./SVGWrapper.css";
+
+export type SVGWrapperProps = SCVWrapperVaraints & {
+  className?: string;
+  viewBox?: string;
+  color?: string;
+  children: ReactNode;
+};
+
+export const SVGWrapper = forwardRef<SVGSVGElement, SVGWrapperProps>(
+  (
+    { className, size, viewBox = "0 0 24 24", color = "black", children },
+    ref
+  ) => {
+    return (
+      <svg
+        ref={ref}
+        className={`${svgWrapper({ size })} ${className}`}
```

Comment:
**suggestion**: what about using `clsx` utility here? 

We could use [clsx](https://www.npmjs.com/package/clsx) to concatenate `classNames`

Reply:
IMO this is enough `className={[svgWrapper({ size }), className].join(" ")}`